keyboards/keebzdotnet/wazowski/config.h

```diff
@@ -20,7 +20,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID 0xFEED
+#define VENDOR_ID 0xKZDN
```

Comment:
This must be a hexadecimal number (0-9, A-F). The defacto standard is to use two ASCII characters instead:
```suggestion
#define VENDOR_ID 0x4B5A // "KZ"
```

Reply:
oops thanks for that catch!